torchacc/utils/patch.py

```diff
@@ -92,3 +92,30 @@ def update_causal_mask(
             return None
 
         LlamaModel._update_causal_mask = update_causal_mask
+
+def patch_qwen():
+    '''
+    Replace search_str with replace_str in `Qwen2FlashAttention2.forward` to avoid xla graph be executed
+    '''
+    import inspect
+    import transformers.models.qwen2.modeling_qwen2 as qwen2
+    # max function will trigger tensor evaluation casuing graph be excuted
+    replace_str = "        rotary_seq_len = kv_seq_len"
+    search_str = "position_ids[:, -1].max().item()"
```

Comment:
Add transformers version checking to ensure this patch works properly. Additionally, print warning information if an unsupported version is detected.

Reply:
fixed